src/dropdown/dropdown.js

```diff
@@ -10,7 +10,7 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
 
   this.open = function(dropdownScope) {
     if (!openScope) {
-      $document.on('click', closeDropdown);
+      document.body.addEventListener('click', closeDropdown, true);
```

Comment:
I think the whole reason he/she switched was to pass in the third parameter to addEventListener...but I could be wrong.